shell/src/main/java/org/crsh/util/Utils.java

```diff
@@ -752,6 +752,8 @@ public static String interpolate(String interpolated, Map<?, ?> context) throws
             status = SCANNING;
           }
           break;
+        default:
```

Comment:
This is superfluous code.  The cases will always be set to one of { ESCAPE, SCANNING, DOLLAR, EVALUATING } and the values are set in this method.  Also as I said above a case in a switch statement should always do something, so "default: break;" is never necessary since the same effect happens by default.